packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -12,6 +12,7 @@ import { IFluidErrorBase, loggerToMonitoringContext } from "@fluidframework/tele
 import {
```

Comment:
It would be super-great if majority of the logic was in shared DocumentDeltaConnection class.
While we will enable it initially in ODSP, we want all services to be able to do so soon after.